students/piotrowski_stanislaw/lesson_06_dicts_tuples_sets_args_kwargs/list_to_dictionary_for_FGI.py

```diff
@@ -0,0 +1,26 @@
+# https://automatetheboringstuff.com/chapter5/
+# List to Dictionary Function for Fantasy Game Inventory
+# piotrsta
+
+from fantasy_game_inventory import display_inventory
+
+
+def add_to_inventory(inventory, added_items):
+    for item in added_items:
+        if item in inventory:
+            inventory[item] += 1
+        else:
+            inventory.setdefault(item, 0)
```

Comment:
`setdefault` jest przydatne, gdy nie wiemy czy taki klucz już jest czy nie, więc skoro sprawdziłeś, że klucza nie ma to wystarczy: inventory[item] = 1, lub zamiast ifów:
`inventory.setdefault(item, 0)` - ustawiamy 0 jeżeli klucza nie ma
`inventory[item] += 1`